app/services/fee_reform/search_offences.rb

```diff
@@ -6,17 +6,19 @@ def self.call(filters)
 
     def initialize(filters = {})
       @filters = filters
+      fee_scheme_array = @filters[:fee_scheme].split(' ')
+      @fee_scheme = FeeScheme.find_by(name: fee_scheme_array[0], version: fee_scheme_array[1])
       @offences_table = Offence.arel_table
       @bands_table = OffenceBand.arel_table
       @categories_table = OffenceCategory.arel_table
     end
 
     def call
-      offences = Offence.unscoped.in_scheme_ten
-                        .joins(offence_band: :offence_category)
-                        .includes(offence_band: :offence_category)
-                        .group('offence_categories.id, offence_bands.id, offences.id')
-                        .order('offence_categories.number, offence_bands.number, offences.description')
+      offences = fee_scheme_offences
+                 .joins(offence_band: :offence_category)
```

Comment:
not a blocker - not you i know, but includes and joins together does not seem right, either bands should be eager loaded with `includes` or not